lib/rspecq/worker.rb

```diff
@@ -185,5 +195,23 @@ def relative_path(job)
     def elapsed(since)
       Process.clock_gettime(Process::CLOCK_MONOTONIC) - since
     end
+
+    # Prints msg to standard output and emits an event to Sentry, if the
+    # SENTRY_DSN environment variable is set.
+    def log_event(msg, level, additional={})
+      puts msg
```

Comment:
Maybe this should be opt-in, using it as a fallback if SENTRY_DSN is not set, as it may not be desirable in some cases?

Reply:
These messages are not displayed to the end user (i.e. not displayed in the reporter), so I think we should keep them until they prove noisy. They definitely help in rspecq development (i.e. when running it locally).